patchwork/app.py

```diff
@@ -203,11 +209,62 @@ def cli(
                 logger.error(f"Config path {config} is neither a file nor a directory")
                 exit(1)
 
+        if debug:
+            inputs["debug"] = True
+
         patchflow_class = find_patchflow(possbile_module_paths, patchflow_name)
         if patchflow_class is None:
             logger.error(f"Patchflow {patchflow_name} not found in {possbile_module_paths}")
             exit(1)
 
+    else:
+        with logger.panel("Initializing Patchwork CLI"):
```

Comment:
This seems duplicated as well whats the difference between the else block and the if block?

Reply:
the difference is using logger.panel in case of debugging disabled and without logger.panel otherwise.

Reply:
If that is the case I think it will be better if the `logger.panel` method can be assigned to a variable instead and is switched to a noop method when debug flag is true.